addon/components/custom-subsidy-form-fields/climate-subsidy-costs-table/edit.js

```diff
@@ -206,6 +233,30 @@ export default class CustomSubsidyFormFieldsClimateSubsidyCostsTableEditComponen
     }
   }
 
+  @action
+  addCustomData() {
+    let uuid = uuidv4();
+    const tableEntryUri = new rdflib.NamedNode(
+      `http://data.lblod.info/id/climate-table/row-entry/${uuid}`
+    );
```

Comment:
TL;DR do we need to store the uuid in the store?

This is were we generate the uri for new custom table entries. The idea is that the custom-row component can then use this uri to check if data already exists and if not, add default values. The quirky thing is that the other table rows all add a uuid triple, but to do that here we would also need to pass the uuid to the component.

This complicates things if we want to fetch the existing custom rows, since we would also need to fetch the uuid values.